mlflow/tracing/fluent.py

```diff
@@ -611,6 +612,10 @@ def predict(input):
             - If not provided and there is no active span, a new span named "Remote Trace <...>"
               will be created and the trace will be merged under it.
     """
+    if not is_trace_enabled():
```

Comment:
note: Additional guardrail so we don't have any similar issue when tracing is disabled